Mathlib/Tactic/ToAdditive.lean

```diff
@@ -798,6 +798,10 @@ def fixAbbreviation : List String → List String
   | "ZSmul" :: s                      => "ZSMul" :: fixAbbreviation s -- from `ZPow`
   | "neg" :: "Fun" :: s               => "invFun" :: fixAbbreviation s
   | "Neg" :: "Fun" :: s               => "InvFun" :: fixAbbreviation s
+  | "unique" :: "Mul" :: s            => "uniqueAdd" :: fixAbbreviation s
+  | "Unique" :: "Mul" :: s            => "UniqueAdd" :: fixAbbreviation s
```

Comment:
I don't think these two lines are needed, toAdditive already does the right thing

Reply:
Eric, you are right.  I got confused and thought that I needed this, but I think that it was a separate issue.  The example below has nothing to do with `UniqueProds`, but seems to confuse `to_additive`:
```lean
import Mathlib.Tactic.ToAdditive

@[to_additive MulA.AddB]  -- to_additive correctly autogenerated target name for MulA.MulB.
    def MulA.MulB : True := trivial
```

Reply:
And, reading the `to_additive` description, I see that `to_additive` is doing what it is programmed to do!

I think that the two extra lines that I had were there because there was a namespace change that `to_additive` was not changing on its own and I expected it to.

Reply:
I'm not sure I completely follow, but if you're happy with them removed, please merge!

bors d+